src/ImageSettings.hs

```diff
@@ -0,0 +1,13 @@
+module ImageSettings
+  ( ImageSettings
+    , imageWidth
+    , imageHeight
+    , fieldOfView
```

Comment:
You're not exporting the constructor.

Reply:
True, I always forget.. Thanks for noticing!